apps/aechannel/test/aesc_test_utils.erl

```diff
@@ -98,13 +98,17 @@ create_tx_spec(InitiatorPubKey, ParticipantPubKey, Spec0, State) ->
       initiator_amount   => maps:get(initiator_amount, Spec),
       participant        => ParticipantPubKey,
       participant_amount => maps:get(participant_amount, Spec),
+      push_amount        => maps:get(push_amount, Spec),
+      channel_reserve    => maps:get(channel_reserve, Spec),
       lock_period        => maps:get(lock_period, Spec),
       fee                => maps:get(fee, Spec),
       nonce              => maps:get(nonce, Spec)}.
 
 create_tx_default_spec(InitiatorPubKey, State) ->
     #{initiator_amount   => 50,
       participant_amount => 50,
+      push_amount        => 20,
+      channel_reserve    => 70,
```

Comment:
This seems like a high number – higher than the initial amounts.
> the minimum amount both peers need to maintain

The docs aren't 100% clear on whether it's _each_ peer or _both together_, but I assume that it's the former.

Reply:
I used the [code](https://github.com/aeternity/epoch/blob/PT-156295688_state-channels_onchaintx_HTTP_API/apps/aechannel/src/aesc_create_tx.erl#L273-L277) as an reference. According to it - this amount should be greater than the initiator.

Reply:
Ah, wrong link - link above was for `push_amount`. The `channel_reserve` must be higher by both of them - (correct link)[https://github.com/aeternity/epoch/blob/PT-156295688_state-channels_onchaintx_HTTP_API/apps/aechannel/src/aesc_create_tx.erl#L279-L288]